scipy/stats/_qmc.py

```diff
@@ -983,9 +1019,27 @@ def __init__(
                            'optimization': optimization}
         super().__init__(d=d, optimization=optimization, seed=seed)
         self.seed = seed
+
+        # important to have ``type(bdim) == int`` for performance reason
```

Comment:
This comment doesn't describe the current state of the following line (the `bdim` values are still numpy scalar types, aren't they?). I do think it might be a good idea to go ahead and do this conversion greedily in the following line (and thus make the comment relevant) and remove the `int()` from later on.

```python
        self.base = [int(bdim) for bdim in n_primes(d)]
```